test/cs_guide_web/controllers/brand_controller_test.exs

```diff
@@ -208,18 +208,24 @@ defmodule CsGuideWeb.BrandControllerTest do
     end
   end
 
+  describe "Get the most common style of the brand" do
+    test "return the most common style for the brand" do
+      brand = %{
+          drinks: [
+            %{drink_styles: [%{name: "style1"}, %{name: "style2"}]},
+            %{drink_styles: [%{name: "style1"}, %{name: "style3"}]}
+          ]
+      }
+      assert CsGuideWeb.BrandController.get_drink_style(brand) == "style1"
+    end
+
+    test "return nil when list of styles is empty" do
+      assert CsGuideWeb.BrandController.get_drink_style(%{drinks: []}) == nil
+    end
+  end
+
   defp create_brand(_) do
     brand = fixture(:brand)
     {:ok, brand: brand}
   end
-
-  defp create_brand(brand_name) when is_binary(brand_name) do
-    params =
-      @create_attrs
-      |> Map.put(:name, brand_name)
-
-    %Brand{}
-    |> Brand.changeset(params)
-    |> Brand.insert()
-  end
```

Comment:
This clause is never used as the one above will always matched with `create_brand(_)`

Reply:
@SimonLab makes sense. 👍